src/action.js

```diff
@@ -38,9 +61,58 @@ export async function run() {
 
   const rwClient = client.readWrite
 
+  let tweetOpts = {}
+
+  if (media?.length) {
```

Comment:
can we extract this block into a function for a better factoring of the code?

Reply:
I moved this block to a dedicate `uploadMedia` function